meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher.service

```diff
@@ -2,11 +2,11 @@
 Description=Resin init flasher service
 Requires=\
     mnt-boot.mount \
```

Comment:
I have the same preference as you, but I am also pro continuity. Maybe @agherzan can give us a decision on the matter.